Assets/MRTK/Core/Utilities/WebRequestRest/Rest.cs

```diff
@@ -17,11 +17,6 @@ namespace Microsoft.MixedReality.Toolkit.Utilities
     /// </summary>
     public static class Rest
     {
-        /// <summary>
-        /// Use SSL Connections when making rest calls.
-        /// </summary>
-        public static bool UseSSL { get; set; } = true;
```

Comment:
While we usually like to avoid breaking changes, it's hard to guarantee (i.e. this is clearly a case where having UseSSL defined on the DevicePortal scope actually makes more sense both from a locality-to-other-SSL-configs, and also because frankly it's not used anywhere in this file).

I was debating in my head if we should keep this around but actually leave the Obsolete attribute on it (i.e. so that it doesn't break people immediately on getting the new upgrade), but actually I think that this is a specific case where both:

1. Unlikely that people are directly touching this given that it's kinda an implementation detail of the device portal feature.
2. I think forcing the error to show up (and also, the solution is super easy) provides a reasonable balance and velocity to getting things fixed for the user.

The only ask here I think is that we add some breaking change notes to ReleaseNotes.md (after this PR goes in: https://github.com/microsoft/MixedRealityToolkit-Unity/pull/8018) similar to how this commit added instructions:

https://github.com/microsoft/MixedRealityToolkit-Unity/commit/204f06de25c9122afd782bb7d60c573e3af340dd

Reply:
Was just thinking through these same things, especially around

>frankly it's not used anywhere in this file

which just seems weird to me. Agreed fully with everything you said.

Reply:
That was my thinking as well. Someone that uses `Rest` to implement another REST API, may stumble upon `UseSSL`. Changing it wouldn't actually have any effect on their REST calls, though as side effect changes the behavior of the `DevicePortal`.

Reply:
Done, added section to `Documentation/ReleaseNotes.md`